app/utils/getUserRank.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import User from "models/User";
+import Rank from "../../shared/utils/rankEnum";
+
+export default function getUserRank(user: User) {
```

Comment:
This could be a `computed` accessor on the user model?

Reply:
Logical 👍🏻